src/transforms/mapperTransforms.ts

```diff
@@ -281,21 +291,25 @@ function transformObjectMapper(pipelineValue: PipelineValue) {
   const { discriminator, discriminatorValue } = objectSchema;
 
   let modelProperties = processProperties(objectSchema.properties, options);
-  const parents = objectSchema.parents ? objectSchema.parents.all : [];
-  const immediateParents = objectSchema.parents
-    ? objectSchema.parents.immediate
-    : [];
+  const parents = getSchemaParents(objectSchema);
+  const immediateParents = getSchemaParents(
+    objectSchema,
+    true /** immediateOnly */
+  );
   const parentsRefs = immediateParents
+    .filter(p => p.type === SchemaType.Object)
```

Comment:
I don't think this filter is needed because it's already being done by `getSchemaParents`.

Reply:
Yes you're right I'll get rid of this one

Reply:
Oh sorry forgot to commit this change, pushing now